client/app/cloud/project/storage/storage-details/storage-details.html

```diff
@@ -1,80 +1,78 @@
-                    <dd class="container-infos-list-item" data-ng-show="url">
-                        <span class="container-infos-list-item-label" data-ng-show="url" data-translate="storage_container_url"></span>
-                        <span class="container-infos-list-item-label word-wrap" data-uib-tooltip="{{ :: 'storage_container_url_info' | translate }}">
-                            {{ url }}
-                            <i class="glyphicon glyphicon-question-sign pointer"></i>
-                        </span>
-                    </dd>
-                    <dd class="container-infos-list-item" data-ng-if="storage.shortcut == 'swift_cname'">
-                            <span class="container-infos-list-item-label" data-translate="storage_static_url"></span>
-                            {{ storage.staticUrl }}
-                            <i class="glyphicon glyphicon-question-sign pointer" data-ng-click="openDNSHelp()" />
-                    </dd>
-                    <div data-ng-if="storage.shortcut === 'pca'" data-ng-include="'app/cloud/project/storage/storage-details/pca-connection-info.html'"></div>
-            </dl>
+    <div class="container-infos rows clearfix" data-ng-if="!loaders.details && !errors.notfound && !errors.notdefined.notdefined && !errors.notfound && !errors.notdefined.notfound">
+        <div class="pull-left" data-ng-if="storage.shortcut">
+            <img class="storage-details-logo" data-ng-src="assets/images/add/storage/{{ storage.shortcut }}.svg" />
         </div>
+        <dl class="container-infos-list" data-ng-class="{ 'container-infos-list_no-icon': !storage.shortcut }">
+                <dd class="container-infos-list-item">
+                    <span class="container-infos-list-item-label" data-translate="storage_nb_objects"></span>{{ objects.length }}
+                </dd>
+                <dd class="container-infos-list-item">
+                    <span class="container-infos-list-item-label" data-translate="storage_container_size"></span>{{ computeStorageSize() | showByte }}
+                </dd>
+                <dd class="container-infos-list-item" data-ng-show="url">
+                    <span class="container-infos-list-item-label" data-ng-show="url" data-translate="storage_container_url"></span>
+                    <span class="container-infos-list-item-label word-wrap" data-uib-tooltip="{{ :: 'storage_container_url_info' | translate }}">
+                        {{ url }}
+                        <i class="glyphicon glyphicon-question-sign pointer"></i>
+                    </span>
+                </dd>
+                <dd class="container-infos-list-item" data-ng-if="storage.shortcut == 'swift_cname'">
+                        <span class="container-infos-list-item-label" data-translate="storage_static_url"></span>
+                        {{ storage.staticUrl }}
+                        <i class="glyphicon glyphicon-question-sign pointer" data-ng-click="openDNSHelp()" />
```

Comment:
You have to use a button, else this action can't be used with a keyboard for example ;)